pkgs/applications/misc/lttoolbox/default.nix

```diff
@@ -0,0 +1,38 @@
+{ fetchFromGitHub, lib, pkgs, stdenv }:
+
+stdenv.mkDerivation {
+  src = fetchFromGitHub {
+    owner = "apertium";
+    repo = "lttoolbox";
+    rev = "v3.7.1";
+    sha256 = "sha256-3lHXKtwQSrMGQEGOGr27e3kB2qKkTFZcEzeAnIm89Rg=";
+  };
+  name = "lttoolbox";
+  buildInputs = [
+    pkgs.autoconf
+    pkgs.automake
+    pkgs.libtool
+    pkgs.libxml2
+    pkgs.libxml2.dev
+    pkgs.icu
+    pkgs.pkg-config
+    pkgs.utf8cpp
+  ];
```

Comment:
You should declare build-time dependencies in `nativeBuildInputs`.

I _suppose_ that `autoconf`, `automake`, `pkg-config`, `utf8cpp`, `libtool` are build-time only.